src/arm/windows/init-by-logical-sys-info.c

```diff
@@ -689,7 +689,7 @@ static bool parse_relation_cache_info(
 		current_cache->flags = CPUINFO_CACHE_UNIFIED;
 	}
 
-	for (uint32_t i = 0; i <= info->Cache.GroupCount; i++) {
+	for (uint32_t i = 0; i < info->Cache.GroupCount; i++) {
 	/* Zero GroupCount is valid, GroupMask still can store bits set. */

```

Comment:
@gaborkertesz-linaro Please, on what processor it can happend that `GroupCount == 0` while the `Cache.GroupMasks[0]` is properly initialized?